### PR TITLE
closes issue #297 Cropped image quality is very low

### PR DESCRIPTION
**Pull request by vinaysajjanapu**
Fixes issue #297 
Changes: 
At line 196 of cropimage class. There the image is getting downsampled by 4.
A scaledbitmap is being created only with those images whose dimensions after downsampling are not even. But this should be performed even if the dimensions are even. I changed that.

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @vinaysajjanapu we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/vinaysajjanapu/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by vinaysajjanapu** [comment]
Changed the sampling rate from 4 to none.
Quality of cropped image significantly increased.
Memory usage increased a bit. around 15-25MB increase for a large image of 6000x4000. That is very less for an image of that size.
I hope this fixes the issue. 
Suggestions are welcome.

File: app/src/main/java/vn/mbm/phimp/me/image/CropImage.java [hunk 1]
```diff
@@ -182,15 +182,11 @@ public void onClick(View v)
...
 
-                BitmapFactory.Options options = new BitmapFactory.Options();
-                options.inSampleSize = 4;
-                mBitmap = BitmapFactory.decodeFile(p[0], options);
-                if (mBitmap.getWidth() % 2 != 0 || mBitmap.getHeight() % 2 != 0) {
-                    //bitmap width , height must even
-                    Log.i("CropImage", "mBitmap width or height not even");
-                    mBitmap = Bitmap.createScaledBitmap(mBitmap, mBitmap.getWidth() * 4, mBitmap.getHeight() * 4, false);
+                /*BitmapFactory.Options options = new BitmapFactory.Options();
+                options.inSampleSize = 4;*/
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/image/CropImage.java]: Please remove commented out code.

Comment (vinaysajjanapu) [inline app/src/main/java/vn/mbm/phimp/me/image/CropImage.java]: done

Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/image/CropImage.java]: Nah the diff is still showing it. Please do look into it. :)

**Pull request merged**